perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/VosManagerBlImpl.java

```diff
@@ -385,6 +408,73 @@ public Vo getVoById(PerunSession sess, int id) throws InternalErrorException, Vo
 		}
 	}
 
+	@Override
+	public List<MemberCandidate> getCompleteCandidates(PerunSession sess, Vo vo, List<String> attrNames, String searchString) throws InternalErrorException {
+		List<ExtSource> extSources = getPerunBl().getExtSourcesManagerBl().getVoExtSources(sess, vo);
+		List<RichUser> richUsers = getRichUsersForMemberCandidates(sess, attrNames, searchString);
+		List<Candidate> candidates = findCandidates(sess, vo, searchString, 0, extSources, false);
+
+		return createMemberCandidates(sess, richUsers, vo, candidates, attrNames);
+	}
+
+	@Override
+	public List<MemberCandidate> getCompleteCandidates(PerunSession sess, Vo vo, Group group, List<String> attrNames, String searchString, List<ExtSource> extSources) throws InternalErrorException {
+		List<RichUser> richUsers = getRichUsersForMemberCandidates(sess, vo, attrNames, searchString);
+		List<Candidate> candidates = findCandidates(sess, group, searchString, extSources, false);
+
```

Comment:
When adding members to group, VO is null and call ``createMemberCandidates()`` ends with null pointer in members manager impl layer. You should re-get VO from a Group and pass it to ``createMemberCandidates()`` since its behavior doesn't differ on VO and is expected non-null.

Reply:
Ok, I have fixed it.